app/js/controllers/dev.js

```diff
@@ -0,0 +1,21 @@
+import React from 'components/react';
+import ReactDOM from 'components/react-dom';
+import { Controller } from 'components/mvc';
+
+import CameraLatestImageView from 'js/views/dev/camera-latest-image';
+
+export default class DevController extends Controller {
```

Comment:
TLDR; Well that doesn't really sound right from pov of taxonomy architecture we'd like to work with and cases that will be related to web notification, but it's not easy to articulate :)

It doesn't feel like we need something related to just data returned from the getter, but more like an independent view with specific purpose - in this case it's photo that has been made by the camera that has been triggered by specific recipe (maybe this view should say that this photo has been taken when door/closed sensor was activated). Next we can have notification that should redirect user to a view that will use more than one getter and ```GetterController``` and URL structure won't scale, I'm not sure if can generalize on this only use case right now. But it requires more time to think about it to make it right.

What should ```latest-image``` in this URL be - is it globally unique getter id (in this case id of the service is redundant, but we still need it to get service name)? Tag? ChannelKind? Some developer-friendly-arbitary-string?

But we need it today/now, so I'm thinking about compromise at the moment...